rpcs3/Gui/SettingsDialog.cpp

```diff
@@ -535,3 +556,53 @@ SettingsDialog::SettingsDialog(wxWindow* parent)
 		config.write(out.c_str(), out.size());
 	}
 }
+
+void SettingsDialog::OnModuleListItemToggled(wxCommandEvent &event)
+{
+	lle_module_list[fmt::ToUTF8(event.GetString())] = chbox_list_core_lle->IsChecked(event.GetSelection());
```

Comment:
This code is executed on item toggle for every single item. The ```wxCommandEvent``` here contains the name and index of the item in the check box, as far as I understood the wx documentation.
I don't think the check box the check box allows toggling multiple items at once, but rather executes the code for every single item toggled.

Qt and wx works different for sure, in Qt I never had any problems with code such this. I didn't noticed any problems while testing. The selections are alwys in sync with the settings, I checked the settings file all the time during my tests.